crates/differ/README.md

```diff
@@ -112,6 +117,42 @@ fn main(){
 }
 ```
 
+Getting the similarity between two words
+```rs
+use differ_rs::{DiffScoreConfig, levenshtein, hamming};
+
+fn main(){
+    let levenshtein_diff = levenshtein("Kittens", "kitten");
+    let hamming_diff = hamming("karolin", "kathrin");
+    let mut config = DiffScoreConfig::default();
+
+    assert_eq!(0.71428573, levenshtein_diff.similarity(&config));
+    assert_eq!(0.5714286, hamming_diff.similarity(&config));
+
+    config.lowercase_sub_cost = 0.5;
+
+    assert_eq!(0.78571427, levenshtein_diff.similarity(&config));
+}
+```
+
+Getting the difference between two words
+```rs
+use differ_rs::{DiffScoreConfig, levenshtein, hamming};
+fn main(){
+    let levenshtein_diff = levenshtein("Kittens", "kitten");
+    let hamming_diff = hamming("karolin", "kathrin");
+    let mut config = DiffScoreConfig::default();
+
+    assert_eq!(1.0 - 0.71428573, levenshtein_diff.difference(&config));
+    assert_eq!(1.0 - 0.5714286, hamming_diff.difference(&config));
+
+    config.lowercase_sub_cost = 0.5;
+
+    assert_eq!(1.0 - 0.78571427, levenshtein_diff.difference(&config));
```

Comment:
To show the percent difference, it's not very ergonomic to show exact decimals here; instead it's more realistic to write it as a 1.0 minus a fraction of two floating points since that's more readable as a user of the library, and also more maintainable